test/performance/odp_ipsecfwd.c

```diff
@@ -489,12 +501,26 @@ static inline uint32_t forward_packets(odp_packet_t pkts[], int num, odph_table_
 			odp_packet_free(pkt);
 			continue;
 		}
-		if (odp_unlikely(!pktio->send_fn(pktio, hash % pktio->num_tx_qs, pkt))) {
-			odp_packet_free(pkt);
+
+		vec = &vecs[pktio->idx];
+		vec->pkts[vec->num++] = pkt;
+		vec->hash = hash;
```

Comment:
This effectively overwrites the flow hash of the preceding packets in the vector, causing them possibly go out through the wrong output queue, which in turn can break packet order in the ordered scheduled mode.

Reply:
Overhauled this in v3.